README.md

```diff
@@ -16,7 +16,7 @@ npm i graphql-metadata
 
 ## Usage
 
-### Annotations parsing
+### [DEPRECATED] Annotations parsing
```

Comment:
In my change I put that on the bottom. Deprecated usage should be on to bottom. Less scrolling. Fixing that now again and moving forward to publish it

Reply:
Thanks, sorry I accidentally force-pushed over your changes.